volttrontesting/platform/test_connection.py

```diff
@@ -40,13 +40,15 @@ def setup_control_connection(request, get_volttron_instances):
 
 
 @pytest.mark.control
+@pytest.mark.xfail
```

Comment:
If you want to mark that **all** the tests as expected to fail, you can mark the test file itself as expected to fail instead of having to mark every individual test. In other words, you can declare a pytestmark global variable and then define it based on whatever pytest.marks you want and they will apply to all the tests. 

```python

import pytest

pytestmark = [pytest.mark.control, pytest.mark.xfail]
``` 

And see this PR as an example in Volttron: #2850 

Ref: https://docs.pytest.org/en/latest/reference/reference.html#globalvar-pytestmark

Reply:
I think we should not use global marks for this....because when we fix one of them we have to put the marks back on the other functions...